pkg/controller/direct/bigqueryreservation/reservation_controller.go

```diff
@@ -129,7 +129,7 @@ func (a *ReservationAdapter) Create(ctx context.Context, createOp *directbase.Cr
 	mapCtx := &direct.MapContext{}
 
 	desired := a.desired.DeepCopy()
-	desiredPb := BigqueryReservationReservationSpec_ToProto(mapCtx, &desired.Spec)
+	desiredPb := BigQueryReservationReservationSpec_ToProto(mapCtx, &desired.Spec)
```

Comment:
It would be great to have a fuzzer when we're changing the mappers on a beta resource